tests/test.c

```diff
@@ -211,6 +212,7 @@ struct test tests[] = {
         _TEST(0059_bsearch, 0, TEST_BRKVER(0,10,0,0)),
         _TEST(0060_op_prio, 0, TEST_BRKVER(0,9,0,0)),
         _TEST(0061_consumer_lag, 0),
+        _TEST(0062_stats_event, 0),
```

Comment:
mark this test as local (not requiring a broker), by setting test flag to TEST_F_LOCAL

You can later run just the local tests with `make run_local`